models/collection.js

```diff
@@ -77,7 +85,8 @@ module.exports = {
         'id',
         'name',
         'pipelineIds',
-        'userId'
+        'userId',
+        'type'
```

Comment:
let's not make the `type` field required
just make it optional so we don't need to do a DB migration

Reply:
I see, thanks for reminding